packages/react-redux-subspace/.babelrc

```diff
@@ -1,3 +1 @@
-{
-  "presets": ["env", "react", "stage-3"]
-}
+{ "presets": ["./.babelrc.js"] }
```

Comment:
Is this a common approach?  I haven't seen many projects with anything other than a [`babelrc` file and an `env` node for variations](https://github.com/reactjs/redux/blob/master/.babelrc#L21) (although they do require [multiple build scripts](https://github.com/reactjs/redux/blob/master/package.json#L24,L25) to seperate them).

Are there any benefits one way or the other?

Reply:
It's an useful hack to construct ur configs with javascript instead of plain JSON. Upcoming babel@7 will have support for `.babelrc.js` by default so you will be able to remove this "proxy" `.babelrc` file

I was always a little bit confused about how merging is done when specifying `env` in `.babelrc` - and this had to memorized and checked each time, when with `.babelrc.js` I just know instantly what's the output config based on javascript conditions, without anything being automated (sometimes unintuitively for me) by babel itself

Reply:
👍